Assets/MixedRealityToolkit.Extensions/LostTrackingService/LostTrackingService.cs

```diff
@@ -3,19 +3,32 @@
 
 using System;
 using UnityEngine;
-using UnityEngine.XR.WSA;
 using Microsoft.MixedReality.Toolkit.Utilities;
 
+#if UNITY_WSA
```

Comment:
Does this mean we would have broken the .NET build if it was on?

Reply:
It wouldn't have, since this will resolve for both IL2CPP and .NET.
It'd probably break an Android or iOS build, but no UWP/WMR builds (I'm slightly surprised it didn't break the Standalone build).

Reply:
Actually, a local Android build succeeded even without this change.
Unity must be doing something to make sure other platforms' namespaces don't break builds (or maybe this script isn't actually being built for some reason)